### DISCUSSION TITLE
Adding an option to lock static offsets

### DISCUSSION THREAD

**Discussion by mintracer** [terjeio/ioSender#325] [Ideas]
Hey!

i was thinking about an option to lock important offsets so they are not changed by stupid behavior :D

My setup is, that i have a manual toolchange at G59.3 (right front corner) so i click in the offset selector the offset G59.3 and start my macro (X0Y0) then it drives to the position... sometimes i forget to switch back to G54 and then the mess begins... i overwrite my G59.3 offset...

So maybe there is a way to lock important static offsets?

thx

**Comment by terjeio** [comment]
Next grblHAL commit will have the option to lock G59.1 - G59.3 via a new setting:

![image](https://github.com/terjeio/ioSender/assets/20260062/4009fb9c-5123-4206-91f4-0838bc712204)

**Reply by mintracer** [reply]
thank you very much! a really usefull feature/option to have!

**Comment by terjeio** [comment]
The change has now been comitted.
